### ISSUE TITLE
Question: How do I know if the training is working or if I should stop and change things?

### ISSUE THREAD

**Issue by TiagoTiago** [7eu7d7/DreamArtist-sd-webui-extension#27]
At which point should I give up on the current training if it's not producing the type of images I want and try changing parameters and starting over, instead of waiting for more training steps on the current training? What should I look for as indication it is or isn't working?

**Comment by 7eu7d7** [comment]
About 5k-8k steps are enough to get good results. If it doesn't work well for more than 10k steps, you can stop training and try to modify the hyper-parameters. For example, increase the number of vectors per token for both positive and negative, or adjust the learning rate and cfg scale.
